pipelines/controller/src/main/resources/templates/index.html

```diff
@@ -14,6 +14,28 @@
           th:src="@{/bootstrap-5.0.2-dist/js/bootstrap.min.js}"></script>
 
   <script>
+    function openLogs(logPath) {
+      var xhr = new XMLHttpRequest();
+      xhr.open("GET", '/download?path='+logPath, true);
+      xhr.send();
+      xhr.onreadystatechange = function() {
+        if (this.readyState === XMLHttpRequest.DONE && this.status === 200) {
+          console.log('Logs fetched.');
+          const blob = new Blob([xhr.response], { type: "" });
+          const a = document.createElement('a');
+          a.setAttribute('target', '_blank');
+          a.href = URL.createObjectURL(blob);
+          a.click();
```

Comment:
Hmm, instead of this trick, can't we just open a new page and show the output of `GET /download...` in it? IOW, if I just open my browser and go to `[ROOT]/download...` I will see the corresponding log file.